geonode/maps/templates/leaflet_maps/map_embed.html

```diff
@@ -65,6 +65,7 @@
             [bbox[1], bbox[0]],
             [bbox[3], bbox[2]]
         ];
+        map.fitBounds(bounds);
```

Comment:
Probably just a missing code.

Reply:
argh, i don't know why i missed this one. 
thanks for the revision.